rllib/core/rl_module/rl_module.py

```diff
@@ -192,3 +199,15 @@ def make_distributed(self, dist_config: Mapping[str, Any] = None) -> None:
     @abc.abstractmethod
     def is_distributed(self) -> bool:
         """Returns True if the module is distributed."""
+
+    def as_multi_agent(self) -> "MultiAgentRLModule":
+        """Returns a multi-agent wrapper around this module."""
```

Comment:
Can you specify more what this would be used for?

To standardize? (and make RLlib always work with MARL, even the env is single agent)?

Reply:
yep. See my answer to jun's comment above.